specs/commandline/inspect.md

```diff
@@ -0,0 +1,105 @@
+
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect or describe all the signatures associated to a signed artifact (image) in a human readable format.
```

Comment:
Need to add an entry for `notation inspect` spec in this document https://github.com/notaryproject/notation/blob/main/specs/notation-cli.md